docs/topics/user_permissions.md

```diff
@@ -0,0 +1,66 @@
+# Users and permissions
+
+
+## What user permissions are defined in IoTP?
+
+In IoTP there are defined Services and SubServices. SubServices exists into Services.
+Tipically a Service represents a smartcity and all of the SubServices represents the verticals of that smartcity. For more details read [multitenancy](../multitenancy.md)
+
+Users are created into Services. The same user name could be used across different Services to represent different Users.
+i.e. "adm1" could exists in "smartcity" Service and "adm1" could be another user for "smartgondor" Service
+
+Roles are created into Services. By default all Services created into IoT Platform are created with the following Roles:
+- ServiceCustomer: Role for a normal user of the Service, with standard capabilities.
+- SubServiceCustomer: Role for a normal user in a SubService, with standard capabilities.
+- SubServiceAdmin: Role for an administrator user in a SubService, with full capabilities.
+
+There is one and unique role common for all Services:
+- Admin: administrator with full capabilities of the Service
+
+Users receive roles assignments into Services (or Subservices). The permissions are determinated by the Role
+which a user has in a service (or subservice). Depending on the IoTP component, user permission implies the ability
+of do some actions or not.
+
+
+| User   | Role               | Service\\```<SubService>```|
+| -------|--------------------|------------------------|
+| adm1   | admin              | smartcity              |
+| adm1   | SubServiceAdmin    | smartcity\\*           |
+| Alice  | SubServiceAdmin    | smartgondor\\palaces   |
+| bob    | SubServiceCustomer | smartcity\\electricity |
+| bob    | SubServiceAdmin    | smartcity\\gardens     |
+
+
+In deep details, each Role in a Service is defined by a Policy for each IoTP component:
+
+- [IoTP Policies](https://github.com/telefonicaid/orchestrator/tree/master/src/orchestrator/core/policies)
+- [Orion component actions](https://github.com/telefonicaid/fiware-pep-steelskin#-rules-to-determine-the-context-broker-action-from-the-request)
+- [Perseo component actions](https://github.com/telefonicaid/fiware-pep-steelskin#-rules-to-determine-the-perseo-cep-action-from-the-request)
+- [Keypass component actions](https://github.com/telefonicaid/fiware-pep-steelskin#rulesKeypass)
+- [Rest components (STH, IOTA) actions](https://github.com/telefonicaid/fiware-pep-steelskin#generic-rest-middleware)
```

Comment:
Suggestion: "REST-based components" or "REST API based components".

Using just "Rest component" (in lowercase) could lead to wrongly think you are referring "to the *rest* of the component" :)